rstblog/builder.py

```diff
@@ -125,12 +128,37 @@ def get_default_template_context(self):
         }
 
     def render_template(self, template_name, context=None):
+        """
+        Render a template in the current context.
+
+        Args:
+            template_name (str): the path to the template you want, relative to
+                the Jinja2 template loader root.
+
+        Keyword Args:
+            context (Context or None): Extra context to pass in to the
+                rendering.
```

Comment:
Yeah; this docstring is in the [Napoleon](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/) format, and that's how you introduce any keyword arguments (i.e. arguments with a `=` in the signature).